man/systemctl.xml

```diff
@@ -342,6 +342,18 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--timeout=</option></term>
+
+        <listitem>
+          <para>Specifies the maximum time to wait for method call completion.
+          If no time unit is specified, assumes seconds. The usual other units
+          are understood, too (ms, us, s, min, h, d, w, month, y). When not
+          specified or when set to 0, the default of
+          <literal>25s</literal> is assumed.</para>
+        </listitem>
+      </varlistentry>
```

Comment:
I wonder if this really should get a primary interface, i.e. be exposed as cmdline parameter. I'd instead suggest to add an env var for it, as we already have more than enough options exposed directly, and this one is probably more on the "debug" side of things... Hence, maybe make this SYSTEMD_BUS_TIMEOUT= instead?

In fact, maybe that env var should be parsed by sd-bus itself, so that it becomes universally supported, not just in systemctl.